yarn.lock

```diff
@@ -13316,6 +13357,21 @@ tar@^4.0.2:
     safe-buffer "^5.1.2"
     yallist "^3.0.3"
 
+telegraf@^4.3.0:
```

Comment:
As I commented earlier, re-run yarn to remove this.

Reply:
Done, thank you